EVA-3176 - Add Release changelog (#401)
EBIvariation/eva-accession: eva-accession-release-automation/publish_release_to_ftp/README_release_changelog.txt

> This document describe technical changes associated with the EVA RefSNP release.

Reviewer: ```suggestion
This document describes technical changes associated with the EVA RefSNP release.
```

Author: Done

EVA RefSNP release 5
====================

- Release is now performed per taxonomy of the submitted variants:

>   Previous release contain variant based on their presence on an assembly.

Reviewer 2: ```suggestion
  Previous releases contained variants based on their presence on an assembly.
```

Author: Done

>   From release 5, a variant will only be present if it was found in the specified taxonomy.

Reviewer 2: ```suggestion
  From release 5, a variant will only be present in a release file if it was found in the specified taxonomy.
```

> - The variants previously marked as multi-mapped have now been deprecated and are available in the deprecated file

Reviewer 2: Suggest summarizing both these categories (multimap and alleles mismatch) as deprecated instead of two separate sentences...

Author: These are coming from two different sources (release under multi-mapped or not released at all), so I think I prefer to keep them separated.

  of each release.
- Previously unreleased variant mismatching the reference genome are now release in the deprecated file of each release.

EVA RefSNP release 4
====================

 - New REMAPPED INFO tag present in the current VCF specifying if the variant is only supported by remapped evidence

EVA RefSNP release 3
====================

 - In this release the variant have been remapped to the latest assembly supported by Ensembl. EVA will endeavour to
   keep the assembly versions in sync with Ensembl versions

EVA RefSNP release 2
====================

 - First release containing data from both dbSNP and EVA

EVA RefSNP release 1
====================

>  - Initial release containing only variant defined in dbSNP and ported to EVA

Reviewer: ```suggestion
 - Initial release containing only variants defined in dbSNP and ported to EVA
```

Author: Done
